packages/payment-integration-api/src/payment-integration-selectors.ts

```diff
@@ -52,6 +52,7 @@ export default interface PaymentIntegrationSelectors {
     getPaymentMethodOrThrow(methodId: string, gatewayId?: string): PaymentMethod;
 
     getShippingAddress(): ShippingAddress | undefined;
+    getShippingAddresses(): ShippingAddress[];
```

Comment:
All look good. Does it make sense to create `getShippingAddressesOrThrow` also?

Reply:
We can, but do we need it? I wanted to just update the core interface as needed only.